Sources/Bow/Data/Either.swift

```diff
@@ -181,6 +181,18 @@ public extension Either {
     public static func eq<EqL, EqR>(_ eql : EqL, _ eqr : EqR) -> EitherEq<A, B, EqL, EqR> {
         return EitherEq<A, B, EqL, EqR>(eql, eqr)
     }
+
+    public static func bifunctor() -> EitherBiFunctor<A, B> {
```

Comment:
Fix `EitherBiFunctor` in all parts of the file. It does not compile now after the class rename.

Reply:
Sorry about that. I got carried away by the Github suggestions